README.md

```diff
@@ -2,15 +2,19 @@
 
 ## Introduction
 
+
 This UI Library provides React components that manage fetching and rendering logic for [Constructor.io's autosuggest services](https://constructor.io/products/autosuggest/).
 
 [Our storybook docs](https://constructor-io.github.io/constructorio-ui-autocomplete) are the best place to explore the behavior and configuration options for this UI Library.
 
 ![Autosuggest](assets/autosuggest-ui.gif)
 
 ## How to use this UI Library
+## Install
 
-There are two main methods for consuming this UI Library in a React project:
```

Comment:
Can we bring this line back?

Reply:
Yep I just thought it's self explanatory